### ISSUE TITLE
Could not find sections and lectures (can't find Videos and Reading Notes section of course)

### ISSUE THREAD

**Issue by psychrophile** [ivan-dyachenko/coursera-downloader#26]
Coursera pages are becoming less straightforward in terms of finding the "https://class.coursera.org/<course>/lecture/index" template pathway.

Is there a way of finding these other than copy-pasting from the urls that do work?

e.g. I'm on course Bioinformatic Methods I

This link does not work because it is not in the correct format
https://www.coursera.org/learn/bioinformatics-methods-1/home/welcome

Intuitively this should work because it replaces <course> with the url part used in the link above:
https://class.coursera.org/bioinformatics-methods-1/lecture/index

Not sure if I'm missing something obvious here- also there is no "Videos and Reading Notes" section.

**Comment by psychrophile** [comment]
Isn't this a problem for anyone else?

Am I missing something here?

**Comment by andreypz** [comment]
Yes, this is a problem for me. Hopefully the code can be updated to deal with this.

**Comment by bani** [comment]
It's a problem for me as well. My course is https://www.coursera.org/learn/analytics-tableau/
